### PR TITLE
fix(input): fix input bug

### PR DESCRIPTION
**Pull request by xiaofeng-bm**
https://github.com/ElemeFE/element/issues/18573

- [ ] Component Migration

Fix #
修复https://github.com/ElemeFE/element/issues/18573

### CODE REVIEW

File: packages/input/src/index.vue [hunk 1]
```diff
@@ -339,7 +339,9 @@ export default defineComponent({
     }
 
     const focus = () => {
-      inputOrTextarea.value.focus()
+      nextTick(() => {
```
Comment (HerringtonDarkholme) [inline packages/input/src/index.vue:342]: Please add reference issue here  // https://github.com/ElemeFE/element/issues/18573

**Comment by Ryan2128** [comment]
Your second commit's scope is error, You may need to change the commit or squash the second commit into the first commit.

**Review by xiaofeng-bm — COMMENTED** [review]
> Your second commit's scope is error, You may need to change the commit or squash the second commit into the first commit.
已改

**Comment by Ryan2128** [comment]
Your change is apply to `Input` component, but your commit message is `fix(button): fix issues`.

The right commit message should be `fix(input): fix issues`

**Review by jw-foss — APPROVED** [review]

**Comment by JeremyWuuuuu** [comment]
LGTM, @HerringtonDarkholme are you ok with this?

**Comment by HerringtonDarkholme** [comment]
> Your change is applied to `Input` component, but your commit message is `fix(button): fix issues`.
> 
> The right commit message should be `fix(input): fix issues`

Please change the commit message.

**Comment by xiaofeng-bm** [comment]
> > Your change is applied to `Input` component, but your commit message is `fix(button): fix issues`.
> > The right commit message should be `fix(input): fix issues`
> 
> Please change the commit message.

改了

**Pull request merged**